aci_interface_policies.tf

```diff
@@ -200,35 +201,37 @@ locals {
 module "aci_leaf_interface_configuration" {
   source = "./modules/terraform-aci-interface-configuration"
 
-  for_each          = { for int in local.new_leaf_interface_configuration : int.key => int if local.modules.aci_interface_configuration && var.manage_interface_policies }
-  node_id           = each.value.node_id
-  module            = each.value.module
-  port              = each.value.port
-  policy_group_type = each.value.policy_group_type
-  policy_group      = each.value.policy_group
-  breakout          = each.value.breakout
-  fex_id            = each.value.fex_id
-  description       = each.value.description
-  shutdown          = each.value.shutdown
-  role              = each.value.role
+  for_each                   = { for int in local.new_leaf_interface_configuration : int.key => int if local.modules.aci_interface_configuration && var.manage_interface_policies }
+  node_id                    = each.value.node_id
+  module                     = each.value.module
+  port                       = each.value.port
+  policy_group_type          = each.value.policy_group_type
+  policy_group               = each.value.policy_group
+  breakout                   = each.value.breakout
+  fex_id                     = each.value.fex_id
+  description                = each.value.description
+  shutdown                   = each.value.shutdown
+  role                       = each.value.role
+  port_channel_member_policy = try("${each.value.port_channel_member_policy}${local.defaults.apic.access_policies.interface_policies.port_channel_member_policies.name_suffix}", "")
```

Comment:
The name_suffix logic should be handled in the local variable `new_leaf_interface_configuration` code.

Reply:
moved to locals for all 3 occurences (interface, subinterface, fexinterface)